pom.xml

```diff
@@ -0,0 +1,45 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Habushu::Cucumber Formatter</name>
+    <description>Formatter for generating Cucumber-format JSON test results using the Behave test framework.</description>
+    <parent>
+        <groupId>org.technologybrewery</groupId>
+        <artifactId>parent</artifactId>
+        <version>8</version>
+    </parent>
+
+    <groupId>org.technologybrewery.habushu.cucumber.formatter</groupId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <artifactId>habushu-cucumber-formatter</artifactId>
+    <packaging>habushu</packaging>
+
+    <build>
+        <directory>dist</directory>
+        <plugins>
+            <plugin>
+                <groupId>org.technologybrewery.habushu</groupId>
+                <artifactId>habushu-maven-plugin</artifactId>
+                <version>2.13.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <pythonVersion>${version.python.default}</pythonVersion>
+                    <managedDependencies>
+                        <packageDefinition>
```

Comment:
S: we probably don't need there here - I think there were likely jsut there as examples in habushu